src/v/ssx/sformat.h

```diff
@@ -26,7 +25,7 @@ struct fmt::formatter<seastar::sstring> final : fmt::formatter<string_view> {
     }
 };
 
-namespace detail {
+namespace fmt::detail {
```

Comment:
@BenPope

Reply:
I think this is a noop since the above `namespace fmt` was removed due to the namespace already being used in the specialization, lgtm

Reply:
not quite true, the `detail` stuff should be in `fmt` as `fmt::detail::is_streamable<>` is used by {fmt} to tell if a type can be formatted using `operator<>`. if we put `is_streamable<>` in `::detail`, {fmt} is not going to use it anymore. and we would suffer from the performance penalty spotted by Ben.

Reply:
makes sense to me. always nice to be more consistent. now that this PR has been approved, shall i just go ahead and update the PR or create anther PR to address this?

Reply:
Just merged, so new PR please.

Reply:
Or, since it's very minor, we just revisit it when fmtlib is updated.